api/types/types.go

```diff
@@ -105,6 +105,7 @@ type Ping struct {
 	APIVersion   string
 	OSType       string
 	Experimental bool
+	Features     map[string]bool
```

Comment:
Unless we want to use this to _disable_ features that are enabled by default, we could simply use a `[]string` (i.e. presence of a feature just means it's in the list?)

Reply:
Yes; Tibor explained that, thanks!